cmd/state-svc/main.go

```diff
@@ -48,13 +48,21 @@ func main() {
 		os.Exit(exitCode)
 	}()
 
+	cfg, err := config.New()
+	if err != nil {
+		// We do not want to log an error here as we want to avoid potential rollbar reports until we load the config
+		logging.Debug("Failed to load configuration: %v", err)
+	} else {
+		logging.CurrentHandler().SetConfig(cfg)
+	}
+
 	logging.SetupRollbar(constants.StateServiceRollbarToken)
 
 	if os.Getenv("VERBOSE") == "true" {
 		logging.CurrentHandler().SetVerbose(true)
 	}
 
-	err := run()
+	err = run()
```

Comment:
Better not to reuse `err` as you'll run into false negatives for `!= nil` if the original type doesn't match. Yeah that's a thing... -.-